### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by JAWD-001**

### CODE REVIEW

**Review by bbelderbos — APPROVED** [review]
Approving this, but having a few questions around relations, can you review?

File: peer_mentoring/groups/models.py [hunk 1]
```diff
@@ -31,6 +29,8 @@ class Group(models.Model):
     description = models.TextField(max_length=250, blank=False, null=False)
     category = models.ForeignKey(Category, on_delete=models.CASCADE)
     added = models.DateTimeField(auto_now_add=True, null=False, blank=False)
+    member = models.OneToOneField(user, on_delete=models.CASCADE)
+    moderator = models.ForeignKey(user, on_delete=models.CASCADE)
```
[the remark this answers is not in the document]
Comment (JAWD-001) [reply peer_mentoring/groups/models.py]: I changed it.

File: peer_mentoring/groups/models.py [hunk 2]
```diff
@@ -31,6 +29,8 @@ class Group(models.Model):
     description = models.TextField(max_length=250, blank=False, null=False)
     category = models.ForeignKey(Category, on_delete=models.CASCADE)
     added = models.DateTimeField(auto_now_add=True, null=False, blank=False)
+    member = models.OneToOneField(user, on_delete=models.CASCADE)
```
[the remark this answers is not in the document]
Comment (JAWD-001) [reply peer_mentoring/groups/models.py]: I changed it

File: peer_mentoring/account_management/templates/user_profile.html [hunk 3]
```diff
@@ -62,4 +62,27 @@ <h5 class="modal-title" id="photo_upload">Update New Photo</h5>
...
+    <small>And some small print.</small>
+  </a>
+  {% endif %}
+
+  {% if comment in recent_comments %}
+  <div class="list-group">
+  <a href="#" class="list-group-item list-group-item-action active" aria-current="true">
+    <div class="d-flex w-100 justify-content-between">
+      <h5 class="mb-1">List group item heading</h5>
+      <small>3 days ago</small>
```
[the remark this answers is not in the document]
Comment (JAWD-001) [reply peer_mentoring/account_management/templates/user_profile.html:82]: I didn't know how to dynamically code it, please tag in next push

File: peer_mentoring/account_management/models.py [hunk 4]
```diff
@@ -38,7 +38,7 @@ class UserProfile(models.Model):
         help_text="Required, please enter a date",
     )
     groups_joined = models.ManyToManyField(Group)
-    # groups_moderated = models.ForeignKey(Group, on_delete=models.CASCADE)
+    groups_moderated = models.ForeignKey(Group, on_delete=models.CASCADE)
```
[the remark this answers is not in the document]
Comment (JAWD-001) [reply peer_mentoring/account_management/models.py]: We do

**Pull request merged**